Assets/HoloToolkit/SpectatorView/Scripts/Recording/ReplayKitRecorder.cs

```diff
@@ -19,63 +19,64 @@ public class ReplayKitRecorder : MonoBehaviour
         /// <summary>
         /// Controls container gameObject
         /// </summary>
-        [Tooltip("Controls container gameObject")] 
+        [Tooltip("Controls container gameObject")]
         [SerializeField]
         private GameObject controls;
 
+#pragma warning disable 0414
         /// <summary>
         /// Seconds to countdown before recording
         /// </summary>
-        [Tooltip("Seconds to countdown before recording")] 
+        [Tooltip("Seconds to countdown before recording")]
         [SerializeField]
         private int countDownNumber = 3;
+#pragma warning restore 0414
```

Comment:
I think using pragma keywords aren't the best solution.

Reply:
do you have a better one?

Reply:
yes, but the countdown delay is/can be set in the editor. I can wrap it in an #if UNITY_IOS block like the other updates, but I though it was more important for the inspector's view of the app to look as close as possible to each other.

Reply:
Causing it to disappear in the Inspector for different build types can also cause lost prefab values as the build types are switched back and forth.

Reply:
I understand that, but we don't do this in other places where we have the same problem.

Reply:
Do you have an example of how it was fixed in another place that would solve this situation? 
If we have cases where things are disappearing from the Inspector, we should reevaluate what we did there. If we have cases where this was solved without `#pragma` and without things disappearing from the Inspector, we should probably put a note in the coding guidelines letting people know our expectations.